Sources/PackageLoading/TargetSourcesBuilder.swift

```diff
@@ -264,7 +278,9 @@ public struct TargetSourcesBuilder {
             let explicitLocalization: String? = {
                 switch rule  {
                 case .processResource(localization: .default):
-                    return defaultLocalization ?? "en"
+                    // FIXME: Restore default localization
```

Comment:
need to be addressed?

Reply:
Yes, I think this is just a draft at this point to get early feedback.  Agree it needs to be addressed before this is merged.